app/models/blueprint.rb

```diff
@@ -228,12 +228,12 @@ def update_catalog_items(the_bundle, catalog_items)
   end
 
   def add_entry_points(new_bundle, entry_points, dialog, service_type)
-    entry_points ||= {
-      'Provision'  => ServiceTemplate.default_provisioning_entry_point(service_type),
-      'Retirement' => ServiceTemplate.default_retirement_entry_point
-    }
+    entry_points = {} unless entry_points
+    entry_points['Provision'] ||= ServiceTemplate.default_provisioning_entry_point(service_type)
```

Comment:
I think this might be doing something subtly different to before....I believe before it would not mutate the argument that's passed in as `entry_points`, and now it can. It may be better to do something like:

```ruby
entry_points = entry_points.dup || {}
entry_points["Provision"] ||= # etc..
```